config/locales/en.yml

```diff
@@ -34,7 +34,8 @@ en:
       training_initiative: &training_initiative Training initiative
       funding_method: &funding_method Funding method
       bursary_applied_for: Bursary applied for
-      no_bursary_applied_for: Not funded
+      scholarship_applied_for: Scholarship applied for
+      no_bursary_applied_for: Not bursary or scholarship funded
```

Comment:
This had been changed to `no_bursary_applied_for: Not funded` (previous line 37) 'Not funded' is the correct new content.